contribute/guidelines.md

```diff
@@ -56,7 +53,7 @@ Due to a `pip` limitation, it is required to increment the major version number
 Pull requests should generally be **reviewed** by someone else than their authors.
 
 This is mandatory for:
-- Any Pull Request on **`openfisca-core`**
```

Comment:
This is indeed moved to the Core `CONTRIBUTING.md`. I don't believe contributors are actually going to read this contributor guidelines thoroughly before contributing, while GitHub provides a strong incentive to read the `CONTRIBUTING.md` file when opening a PR, so it makes more sense IMO to move it there. See https://github.com/openfisca/openfisca-core/pull/497.